packages/babel-plugin-transform-object-rest-spread/src/shouldStoreRHSInTemporaryVariable.ts

```diff
@@ -12,7 +12,10 @@ export default function shouldStoreRHSInTemporaryVariable(
 ): boolean {
   if (!node) return false;
   if (node.type === "ArrayPattern") {
-    const nonNullElements = node.elements.filter(element => element !== null);
+    const nonNullElements = node.elements.filter(
```

Comment:
Would it be possible to add a test for this change, using this plugin together with just syntax support for void patterns?